todo_app/data/trello_items.py

```diff
@@ -0,0 +1,95 @@
+import requests,os, json
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
```

Comment:
We shouldn't need this command since flask will do it automatically if the dotenv library exists, and warning it might catch you out in future modules when we look at testing! Similarly for another comment below, for a full explanation:
* The tests will import this file at some stage
* When they do, this code will _immediately_ run, so:
  * If it happens early, it might cause you to load the wrong (live, not test) variables in for something important
  * If it happens late, it might override the testing variables

So better to remove this bit

Reply:
Hi Jack, I will look at making these changes as for some reason it was not working for me & I probably took the lets make this work approach & deal with the issues later :).